src/js/components/Article.js

```diff
@@ -491,6 +513,11 @@ export default class Article extends Component {
       controls = this._renderControls();
     }
 
+    let pagination;
+    if (this.props.pagination && this.state.atBottom && 'row' === this.props.direction) {
```

Comment:
I believe that `this.state.atBottom` is not needed here.

If the page/chapter does not have a scroll, atBottom will never be true